app/(main)/league/[leagueId]/entry/all/page.tsx

```diff
@@ -29,9 +33,34 @@ const Entry = ({
 }): JSX.Element => {
   const [currentWeek, setCurrentWeek] = useState<IGameWeek['week']>(1);
   const [entries, setEntries] = useState<IEntry[]>([]);
+  const [leagueName, setLeagueName] = useState<string>('');
   const [loadingData, setLoadingData] = useState<boolean>(true);
+  const [survivors, setSurvivors] = useState<number>(0);
+  const [totalPlayers, setTotalPlayers] = useState<number>(0);
   const { user } = useDataStore((state) => state);
 
+  /**
+   * Fetches the current league name.
+   * @returns {Promise<void>}
+   */
+  useEffect(() => {
+    /**
+     * Fetches the current league name.
+     */
+    const getCurrentLeagueName = async (): Promise<void> => {
+      try {
+        const league = await getCurrentLeague(leagueId);
+        setLeagueName(league.leagueName);
+        setSurvivors(league.survivors.length);
+        setTotalPlayers(league.participants.length);
+      } catch (error) {
```

Comment:
I feel like this should have a `throw` statement if it errors out.

Reply:
@choir27 updated in latest commit.